src/js-display-panel.ts

```diff
@@ -0,0 +1,135 @@
+import { ToggleStatus } from './js-feature-toggle-manager';
+import { QueryStringReader } from './github/gtmsportswear/js-utilities@1.0.0/js-utilities';
+
+export abstract class DisplayPanel {
+  public draw(): Element {
+    const panel = document.createElement('div');
+    panel.classList.add('display-panel');
+
+    return panel;
+  }
+}
+
+export class FeatureToggleDisplayPanel extends DisplayPanel {
+  constructor(private toggles: ToggleStatus[]) {
+    super();
```

Comment:
Why does this need to call the parent constructor? Is it just good practice, or is it necessary even with a default constructor?